apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -125,8 +125,7 @@ export default class ModerationNotificationList extends Component {
                   <ModerationNotification
                     key={i}
                     apiUrl={this.props.moderationCommentsApiUrl + item.pk + '/'}
-                    classifications={Object.keys(item.category_counts)}
-                    classificationsCount={Object.values(item.category_counts)}
+                    classifications={Object.entries(item.category_counts).map(([k, v]) => `${k}: ${v}`)}
```

Comment:
maybe we could shift this into `ModerationNotification`, so we could style the number separately from the classificationtext. Its a tiny detail though. Leave it to you to decide if you want to touch this again

Reply:
okay, this effort does not justify the benefit. It is working fine as it is.